docs/source/contributing/developing.adoc

```diff
@@ -194,11 +193,11 @@ $go test -v <relative path of package> -run "Test<Regex pattern to match tests>"
 For more information about test options, run the `go test --help` command and review the documentation.
 
 [[integration-tests]]
-=== Running integration tests
+=== Running Integration Tests
 
-Integration tests utilize https://github.com/DATA-DOG/godog[`godog`], which uses Gherkin (Cucumber) to define test cases.
-The test cases are defined in *_test/integration/+*+.feature_*.
-Per default, the tests are executed against the binary created by `make build`, that is *_$GOPATH/bin/minishift_*.
+Integration tests utilize link:https://github.com/DATA-DOG/godog[Godog], which uses Gherkin (Cucumber) to define test cases.
+The test cases are defined in *_test/integration/features_*.
```

Comment:
I take this back. The directory does exist. I believe this got refactored at some stage. The idea with *.feature was to make explicit what the test files are, but I guess we don't need it.